### PR TITLE
New Changeling Mutation: Fleshy Maul

### PR DESCRIPTION
**Pull request by robustinII**
## What Does This PR Do
adds new mutation weapon for changeling - fleshy maul. it's pretty good against armored targets and good for bone breaking.
ported from old rupara

Damage 10
Armor Pierce 80 %
DNA 4

Increased bone break chance.
Instakills little bots (like beepsky), pais and drones
Can break any door: airlock, shutters, classic.
Effective against structures.

(no throws or knockbacks or knockdowns like on video. outdated)

## Why It's Good For The Game
the option between two weapons

## Images of changes

https://github.com/ParadiseSS13/Paradise/assets/164400188/06e95c4d-cdb5-49e6-9d0d-c8d6d5356b59

## Testing
everything in the video above lol

## Changelog
:cl:
add: fleshy maul action icons, in hands and mutation
/:cl:

### CODE REVIEW

**Comment by Komrad822** [comment]
Please, in the pr description, put the stats of the maul, what it does, how much it cost, etc.

**Comment by spacegeese** [comment]
sick, is it able to break doors to compensate for not being able to pry them open?

**Comment by robustinII** [comment]
> sick, is it able to break doors to compensate for not being able to pry them open?

of course.

**Comment by 1080pCat** [comment]
> Effective against structures.

does this mean it can break walls?

**Comment by robustinII** [comment]
> > Effective against structures.
> 
> does this mean it can break walls?

yep, everyithing in da code

File: code/modules/antagonists/changeling/powers/mutations.dm [hunk 1]
```diff
@@ -173,6 +174,107 @@
...
+	REMOVE_TRAIT(src, TRAIT_FORCES_OPEN_DOORS_ITEM, ROUNDSTART_TRAIT)
+
+/obj/item/melee/arm_blade/fleshy_maul/attack_self(mob/user)
+	..()
+	if(swing)
+		swing = FALSE
+	else
+		swing = TRUE
+
+	to_chat(user, "<span class='changeling'>We ready to [swing ? "swing at" : "crush"] our prey!</span>")
```
[the remark this answers is not in the document]
Comment (robustinII) [reply code/modules/antagonists/changeling/powers/mutations.dm:217]: it's for enable and disable for throwing targets

**Comment by Somniworld** [comment]
I feel like this is a bit overtuned. Despite the low force, you can push people into walls, which deals a lot of brute damage. This is in addition of a 2 tile range , akin to the monofilament implant. I believe this makes it a straight up superiour upgrade to the armblade (which is already rather strong!), as long as you're mindful of your environment. 

I'd like to see the balance team's take on this, but I feel like either the 2-tile range or the knockback into walls needs to go for this to not be horrendous to fight.

**Comment by Stoniest** [comment]
This will make armblade almost completely obsolete. Balance wise, from what it originally was, it is in a much better state then before, however, a weapon this strong should have some kind of downside. I'd suggest removing knockdown & knock-back in exchange for an ability that'll do that instead and basically break the weapon, putting it into a type of cooldown state.

Changeling does need a bit more variety in the terms of weapon options, but also I believe the design team wants to move away from so much combat focus stuff for changeling or at least have them changed to help compliment the core idea of what a changeling is, a stealth antag. With that being said, the current state of the PR is that this weapon is still overtuned.

**Comment by dudley500** [comment]
It being a non parriable melee weapon with knockdown and knockback makes it too powerful. I am opposed to it as it stands. Stoniests suggestions are something to consider forsure.

**Comment by chuga-git** [comment]
Hi there, sorry for the delay in feedback these last weeks. I've finally gotten around to taking a good look at this ability. After playing around with it on a local build, I don't think this can be put into the game in its current state.

It might do 10 damage, sure, but it also quickly destroys any structure in the game. That means doors and walls. There are very few things in the game that have the capability to do this - the ones that do are limited by some kind of drawback. Fleshy maul does not have any drawbacks that would balance its unlimited ability to allow a Changeling to Kool-aid man through the station. 

I really like the concept, but as it stands, the design is too fundamentally flawed for tweaks to fix it. I do encourage you to iterate on the design in the future, though.

**Comment by S34NW** [comment]
This PR has failed the vote and will be closed.
1 Sprite team objection.
5 Balance team objections.
2 Design team objections.

Some internal comments include (anonymised):

- Still against the idea of this in its base design even after the nerfs to it.
- As much as I want this, I don't think this design is saveable. I'm convinced that there was little thought put into actually balancing this. 10 damage on hit is not enough of a substitute for being able to quickly destroy walls, reinforced walls, doors, windows, etc. This is like an RCD on crack. I like the idea, but the implementation is so flawed that it needs to be redesigned from the ground up.
- comment on PR
- This will make armblade almost completely obsolete. Balance wise, from what it originally was, it is in a much better state then before, however, a weapon this strong should have some kind of downside. I'd suggest removing knockdown & knock-back in exchange for an ability that'll do that instead and basically break the weapon, putting it into a type of cooldown state.
- Until that 2 tile range or the knock back is removed, I will stay on the stance that it has bloated utility. They also recently made it unparriable. 5/29 - Still feel its overloaded, armblade is a no-drop one handed fireaxe and thats it. Its all combat and tentacle is all utility.Feel that if this was changed to an AOE dispacement ability (variable garg stomp) itd be much better
- I want the balance team to look this over before I feel like it can be approved
- icon needs some tweaking before it can get merged
- Structure breaking bad. Echoing above.

**Pull request closed without merging**